hack/app_sre_create_image_catalog.sh

```diff
@@ -19,67 +18,33 @@ git clone \
     https://app:${APP_SRE_BOT_PUSH_TOKEN}@gitlab.cee.redhat.com/service/saas-hive-operator-bundle.git \
     $SAAS_OPERATOR_DIR
 
-# remove any versions more recent than deployed hash
```

Comment:
I don't think we need this for hive. It cannot be rolled back. If something is deployed and we need an earlier commit, the current commit should be reverted in a new commit. Please correct me if I'm missing something.

Reply:
So this exists to for production deploys only see [here](https://github.com/openshift/hive/blob/master/hack/app_sre_build_deploy.sh#L31). What it does is removes any versions in front of the current deployed hash so that when you this new CRD is push it "replaces" the last deployed version to prod. Stage pushes all versions, production only pushes those we promote in saas-hive. I believe you still want this unless you are pushing every version to prod.

Reply:
+1 to what @jharrington22 said